Not a bug just a suggestion for tests

Hello.

First of all thanks for this great package ❤️ 

This is not a huge issue just a matter of form. When pulling this package in my project my IDE (PHPStorm) freaks out because it says that there're multiple definition of the `test` namespace because you have a generic namespace inside `fixtures/repository/tests` which does not follow your others tests namespace criteria like rest of the tests suite that use `namespace Lorisleiva\LaravelDeployer\Test;`

I think the solution is fairly easy and I would fix that if you don't have a plan for this.

Other than this, this package is extremely well made.

--- a/.gitattributes
+++ b/.gitattributes
@@ -1,6 +1,0 @@
-* text=auto
-
-/.gitignore export-ignore
-/phpunit.xml export-ignore
-/tests export-ignore
-/.gitattributes export-ignore

--- a/composer.json
+++ b/composer.json
@@ -41,5 +41,13 @@
                 "Lorisleiva\\LaravelDeployer\\LaravelDeployerServiceProvider"
             ]
         }
+    },
+    "archive": {
+        "exclude": [
+            "/docs",
+            "/tests",
+            "/composer.lock",
+            "/phpunit.xml"
+        ]
     }
 }

